### ISSUE TITLE
About the training data mask

### ISSUE THREAD

**Issue by HaiyangLiu1997** [vita-epfl/openpifpaf#151]
Hi, Sven! it's me again! today I have a interesting question want to consult, but It not related to model self but dataloader, I hope it doesn't bother you.
 
After reading some reproduce code in the pose estimation field, I found that there usually three different methods to generate the training data. 

1. check the total number of keypoints label in one image is > 0 to do the first fliter, similar to use "person" catalog by coco. **Then mask the person which size small, too close, keypoints ann < 5 or crowd.** after these, each person will generate one meta to train, so totally about 110K imgs to train for coco train2014 dataset.

2. using the same method as 1, but one image only using one time, so totally 55K imgs for train2017

3. your method, I found it seems you didn't do any special mask such as ** mask the person which size small, too close, keypoints ann < 5 or crowd.** the mask valid area function seems just mask the invalid area generate by data augment.

I have test your method and the first method to train a openpose model, just change the dataloader, for some middle result in 30 epochs, the single scale mAP of your method is 40.9 in val2017, the first one is 44.3%. 

Of course this doesn't the finial result and I used the same hyper-parameters(adam 1e-5 no weight decay, batchsize16) after changing the dataloader, so these test may not so meaningful, however, I also want to ask have you ever test these different data generate method in your model? and do you have some commits about which method is better?

**Comment by HaiyangLiu1997** [comment]
supplement: the result for larger data loader is from 12 epoch, has the similar iterations number

**Comment by jialee93** [comment]
Hi @HaiyangLiu1997, your issue is interesting and critical and thx for sharing. Could you give more comparisons and results about the three data preparation?
